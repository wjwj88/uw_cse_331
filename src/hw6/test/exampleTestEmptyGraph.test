# Test the parser and BFS on empty file  generic version-Jun Wang 5:00

LoadGraph empty emptyGraph.tsv

ListNodes empty

FindPath empty node_A1 node_B1