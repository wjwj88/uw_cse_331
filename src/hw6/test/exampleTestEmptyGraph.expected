# Test the parser and BFS on empty file  generic version-Jun Wang 5:00

loaded graph empty

empty contains:

unknown character node A1
unknown character node B1